other-docs/guides/upgrading/v10.md

```diff
@@ -88,6 +88,6 @@ The latest version of ElasticPress introduces lots of bug fixes and enhancements
 - The ability to index comments, this is disabled by default but can be enabled via your config
 - New CLI commands to help with debugging:
   - `wp elasticpress get-mapping` shows the current mapping configuration for the specified indexes
-  - `wp elasticpress request` allows you to execute arbitrary requests from the command line to aid with
+  - `wp elasticpress request` request allows you to execute arbitrary requests from the command line to aid with debugging or maybe just wp elasticpress request allows you to execute arbitrary requests from the command line
```

Comment:
Would have preferred suggestions or discussions be on the issue rather than in commits.

```suggestion
  - `wp elasticpress request` allows you to execute arbitrary requests from the command line to aid with debugging
```

Reply:
Oversight from my side, thanks for this catch @shadyvb!